spec/support/helpers/api_helper.rb

```diff
@@ -30,4 +32,20 @@ def build_response(body: {}, http_status_code: 200)
   def build_encoded_token(id:, secret:)
     Base64.strict_encode64("#{id}:#{secret}")
   end
+
+  def read_stub_file
```

Comment:
this is a bit vague... what is a "stub file"? probably should be "fixture". the menu api response files is hardcoded in the method.. either pass that in as a method param or consider calling this `read_menu_api_response`

Reply:
renamed the method.